### ISSUE TITLE
[0.13.16] scalaVersion not working on sbt 0.13.16-M1

### ISSUE THREAD

**Issue by eed3si9n** [sbt/sbt#3165]
## steps

https://github.com/scala/community-builds/pull/535

## problem

```
Cyclic reference involving
*:scalaVersion
root-950fabd31421f835198eabd03f5fa1753af44c91/*:crossScalaVersions
root-950fabd31421f835198eabd03f5fa1753af44c91/*:scalaVersion
```

**Comment by jvican** [comment]
Shouldn't the milestone be 0.13.16 instead of 1.0.0-M6?

**Comment by eed3si9n** [comment]
I'm putting this in "1.0.0-M6" to have a single todo list for the period.

**Comment by eed3si9n** [comment]
I think the root cause here is actually AutoPlugin ordering that @dwijnand fixed in scala/sbt-scala-modules#20 so I'm closing this for now.
